stable/mission-control/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: mission-control
 description: A Helm chart for JFrog Mission Control
-version: 2.0.2
+version: 2.1.0
```

Comment:
Why does this justify a minor version bump? I think this should be 2.0.3

Reply:
Done